app/styles/ui/_acknowledgements.scss

```diff
@@ -3,7 +3,7 @@
 
   .dialog-content {
     height: 300px;
-    overflow: scroll;
+    overflow: auto;
```

Comment:
How about setting `overflow-x: hidden` and `overflow-y: scroll`? That’ll remove the horizontal scrollbar.

Reply:
@cwongmath we can set `overflow` to two values, as per [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow):

> The `overflow` property is specified as one or two keywords chosen from the list of values below. If two keywords are specified, the first applies to `overflow-y` and the second to `overflow-x`. Otherwise, both `overflow-x` and `overflow-y` are set to the same value.

So it could be:

```css
overflow: scroll hidden;
```

or this:

```css
overflow-y: scroll
overflow-x: hidden
```

I'm slightly leaning towards the first option over the second, but the order of `-x` and `-y` might be confusing at a glance. Also, I'm not sure how we settled on `scroll` over `auto` here, but I'll defer to @j-f1 and @outofambit as they've been following this PR more closely than I have.

Also, can we ensure we do the same fix in `app/styles/ui/dialogs/_release-notes.scss` which was the original example of the issue?

Reply:
good shout @shiftkey. we should use `auto` over `scroll`. i'm also fine with a single line `overflow`. @cwongmath how does that sound?

Reply:
:ok_hand: